src/main/java/net/conjur/api/clients/AuthnClient.java

```diff
@@ -31,10 +32,10 @@
 
     private String apiKey;
 
-    public AuthnClient(final Credentials credentials, final Endpoints endpoints) {
+    public AuthnClient(final Credentials credentials, final Endpoints endpoints, final SSLContext sslContext) {
```

Comment:
Same here too - I think we would want the constructor to be able to be created without the SSL Context.

Reply:
Fixed.